mobile/native/src/db/mod.rs

```diff
@@ -35,7 +35,7 @@ pub const MIGRATIONS: EmbeddedMigrations = embed_migrations!();
 /// Sets the number of max connections to the DB.
 ///
 /// This number was arbitrarily chosen and can be adapted if needed.
-const MAX_DB_POOL_SIZE: u32 = 16;
+const MAX_DB_POOL_SIZE: u32 = 1;
```

Comment:
Nit: now the value is not arbitrarily chosen anymore. Maybe we can adapt the comment?